aaq/views.py

```diff
@@ -222,3 +223,21 @@ def search(request, *args, **kwargs):
     }
 
     return Response(json_msg, status=status.HTTP_200_OK)
+
+
+@api_view(("POST",))
+@renderer_classes((JSONRenderer,))
+def check_urgency_v2(request, *args, **kwargs):
+    serializer = UrgencyCheckV2Serializer(data=request.data)
+    serializer.is_valid(raise_exception=True)
+    message_text = serializer.validated_data["message_text"]
+    url = urllib.parse.urljoin(settings.AAQ_V2_API_URL, "/urgency-check-v2")
+    headers = {
+        "Authorization": settings.AAQ_V2_API_URL,
+        "Content-Type": "application/json",
+    }
+
+    response = requests.request("POST", url, json=message_text, headers=headers)
+    is_urgent = {"is_urgent": response.json()["is_urgent"]}
```

Comment:
We could just change this to:
`return Response(response.json(), status=status.HTTP_200_OK)`